bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/values/MapValueImpl.java

```diff
@@ -361,7 +361,7 @@ public String stringValue() {
                             continue;
                         }
                         String fieldName = field.getKey();
-                        V fieldVal = get((K) fieldName);
+                        V fieldVal = get(fieldName);
                         sj.add(fieldName + ":" + getStringValue(fieldVal));
```

Comment:
replace : with =

Reply:
Do we need this? now the object are represented with ObjectValues, not by maps

Reply:
@SupunS wdyt ?

Reply:
As per the offline chat with @SupunS  we can remove that..